Source/Blazorise/Interfaces/IClassProvider.cs

```diff
@@ -499,6 +499,8 @@ public interface IClassProvider
 
         string ListGroupItemDisabled();
 
+        string ListGroupItemColor( Color color );
```

Comment:
Don't forget to add implementation to all providers.

Reply:
Ok